examples/client/client.c

```diff
@@ -106,14 +106,14 @@ static int NonBlockingSSL_Connect(WOLFSSL* ssl)
     error = wolfSSL_get_error(ssl, 0);
     sockfd = (SOCKET_T)wolfSSL_get_fd(ssl);
 
-    while (ret != SSL_SUCCESS && (error == SSL_ERROR_WANT_READ ||
-                                  error == SSL_ERROR_WANT_WRITE ||
+    while (ret != WOLF_SSL_SUCCESS && (error == WOLF_SSL_ERROR_WANT_READ ||
```

Comment:
wolfSSL is one word.  Please remove underscore between WOLF and SSL.  Otherwise looks good.